.pre-commit-config.yaml

```
@@ -54,7 +54,7 @@ repos:
     exclude: .github/labeler.yml|.pre-commit-search-and-replace.yaml
 
 - repo: https://github.com/mattlqx/pre-commit-search-and-replace
-  rev: v1.0.5
+  rev: v1.1.3
```

```suggestion
  rev: v1.0.5
```
Nevermind, it seems to be timing out during the installation phase...